README.md

```diff
@@ -69,7 +69,7 @@ However, this starts (and afterwards stops) docker containers. This includes sta
  adding a startup time of several minutes. To circumvent this you also run the test on already running containers with
  for example:
  ```bash
-./gradlew -x composeUp -x composeDown :apix-integrationtests:test-61:integrationTest -Pprotocol=http -Phost=localhost -Pport=8061
+./gradlew -x composeUp -x composeDown :apix-integrationtests:alfresco:74:integrationTest -Pprotocol=http -Phost=localhost -Pport=8074
```

Comment:
Weren't ports now all set to 8080, or I"m mixing things up?

Reply:
Yes, but that argument is still available to use.